tests/spec/modules/browser/runtime.spec.js

```diff
@@ -0,0 +1,28 @@
+import {
+  sendMessageToBackground,
+  addOnMessageListener
+} from '../../../../src/modules/browser/runtime';
+
+describe('runtime module:', () => {
+  describe('sendMessageToBackground:', () => {
+    it('return a promise when is called', () => {
+      expect(sendMessageToBackground('string msg') instanceof Promise).toBe(
+        true
+      );
+    });
+  });
+
+  describe('addOnMessageListener:', () => {
+    it('no happend when is called with a listener arg', done => {
```

Comment:
__[Misspell]__ <sub>posted by Sider</sub>

"happend" is a misspelling of "happened"